README.md

```diff
@@ -1,2 +1,127 @@
+
+Each Contributor status has 3 levels of Github access, in the order of increasing access:
+
+1. Triage
+
+    Requirement: One voucher from anyone with level 2 or above access.
+
+2. Write
+
+    Requirement: One voucher from anyone with level 3 access or two from
+anyone with level 2 access. If only vouched by those with level 2 access, the
+user must have Triage access to be promoted.
+
+3. Maintain
+
+    Requirement: One voucher from anyone with level 3 access. This permission
+is usually reserved for those maintaining the respective repositories.
+
+Each of the levels implies having the previous levels - e.g. level 2 implies
+level 1. A contributor with Write access will be able to triage pull requests
+in accordance with Github access policies.
+
+## Detailed Design
+
+### Level 1 - Triage Access
+
+Requirements: One voucher from any user with level 2 or above access.
+
+Contributors with Triage access will have the associated [triage Github access
+policy](https://docs.github.com/en/organizations/managing-user-access-to-your-organizations-repositories/repository-roles-for-an-organization#permissions-for-each-role)
+for the [Solana Program Library](https://github.com/solana-labs/solana-program-library) (SPL) repository or the 
+[Solana](https://github.com/solana-labs/solana) monorepo.
+
+### Level 2 - Write Access
+
+Requirements: One voucher from anyone with level 3 access or two vouchers from
+anyone from level 2. Applicants must have level 1 access to apply for level 2.
+
+Contributors with Write access will have the associated [write Github access
+policy](https://docs.github.com/en/organizations/managing-user-access-to-your-organizations-repositories/repository-roles-for-an-organization#permissions-for-each-role)
+for the [Solana Program Library](https://github.com/solana-labs/solana-program-library) (SPL) repository or the 
+[Solana](https://github.com/solana-labs/solana) monorepo.
+
+Most notably these permissions allow the contributor to approve and merge pull
+requests.
+
+### Level 3 - Maintain Access
+
+Requirements: One voucher from anyone with level 3 access.
+
+Contributors will have the associated [maintain Github access
+policy](https://docs.github.com/en/organizations/managing-user-access-to-your-organizations-repositories/repository-roles-for-an-organization#permissions-for-each-role)
+for the [Solana Program Library](https://github.com/solana-labs/solana-program-library) (SPL) repository or the 
+[Solana](https://github.com/solana-labs/solana) monorepo.
+
+Users with Maintain access are responsible for managing the the repositories and
+granting user access based on the above requirements.
+
+### Vouching Process 
+
+To receive a Contributor status and the respective level of access as defined in the Contributor Access Policy, follow these steps:
+
+1. Open an issue on the [Solana Program Library](https://github.com/solana-labs/solana-program-library) (SPL) repository 
```

Comment:
For opening the request issue, using this repo seems better than directly in the monorepo or SPL.